src/sys/epoll.rs

```diff
@@ -80,6 +88,13 @@ pub fn epoll_create() -> Result<RawFd> {
 }
 
 #[inline]
+pub fn epoll_create1(flags: EpollFdFlag) -> Result<RawFd> {
+    let res = unsafe { ffi::epoll_create1(flags.bits() | EPOLL_CLOEXEC.bits) };
```

Comment:
- The way you are adding the default flag is actually enforcing it, therefore making it impossible for someone to call `epoll_create1` without the `EPOLL_CLOEXEC` flag.
- I don't think that any of the flags should be turned on by default, especially not without documentation. (Principle of Least Surprise)